zerver/models/users.py

```diff
@@ -78,6 +78,19 @@ class UserBaseSettings(models.Model):
     COLOR_SCHEME_CHOICES = [COLOR_SCHEME_AUTOMATIC, COLOR_SCHEME_NIGHT, COLOR_SCHEME_LIGHT]
     color_scheme = models.PositiveSmallIntegerField(default=COLOR_SCHEME_AUTOMATIC)
 
+    # Information density is established user by user through
+    # adjustments to the font size, line height, and full line
+    # space (the amount of space between one block element and
+    # another, expressed as a percentage of the line height).
+    WEB_FONT_SIZE_PX_LEGACY = 14
+    WEB_LINE_HEIGHT_PX_LEGACY = 20
+    WEB_FULL_LINE_SPACE_PERCENT_DEFAULT = 100
```

Comment:
Hmm, what's the thinking on having an integer with units of percent? I feel like it's hard for that to not introduce rounding complexity. What are some sample values of that amount of space that we might find ourselves using? I wonder whether just using a value in pixels and imputing the percent from that would be more coherent.

(I'm also not totally sure I understand why this should be an independent setting parameter)

Reply:
Hmmmmm. Yes, you're right, a pixel value--probably initially defaulting to the same value as the line height--is the better move here.

The reason this is important to have as an additional, independent setting is that someone who prefers to skim through a topic might well want to have a very tight line height--more text per vertical span of space on screen. BUT. They might also want a more exaggerated space between blocks, so that it's easy for their eyeballs to get to the next chunk and continue skimming.

By contrast, someone else might want a nice comfortable line height for easy reading (for them), and only need the slightest indicator of the space between blocks.